src/shallow/init.lua

```diff
@@ -0,0 +1,21 @@
+local createReconciler = require(script.Parent.createReconciler)
+local Type = require(script.Parent.Type)
+local RobloxRenderer = require(script.Parent.RobloxRenderer)
+local ShallowWrapper = require(script.ShallowWrapper)
+
+local robloxReconciler = createReconciler(RobloxRenderer)
+
+local shallowTreeKey = "RoactTree"
+
+local function shallow(element, options)
+	assert(Type.of(element) == Type.Element, "Expected arg #1 to be an Element")
+
+	options = options or {}
+	local maxDepth = options.depth or 1
+
+	local rootNode = robloxReconciler.mountVirtualNode(element, nil, shallowTreeKey)
```

Comment:
I'm imagining most tests will be inserting the shallow logic in between calls to mount and unmount. Because of this, we should just take the tree as an argument, and figure out how to get access to the root node